readme.md

```diff
@@ -57,15 +58,19 @@ Select 1:
 Create a variable and store the string "pizza" in it
 
 ```js
-answer goes here
+
+var food = ("pizza")
+
 ```
 
```

Comment:
this is technically correct, but the parens don't do anything here. typically it'd be written out `var food = "pizza"`